src/main/java/com/ss/training/lms/controller/AdminBranchController.java

```diff
@@ -13,5 +20,22 @@
 
 	@Autowired
 	AdminBranchService service;
+	
+	/**
+	 * @return
+	 */
+	@RequestMapping(path = "/lms/admin/branch")

```

Comment:
path should be /lms/admin/branches for all branches. for single branches the path will be /lms/admin/branches/{branchId}

Reply:
I thought Akbar said the plural form should only be used when followed by an identifier to select one or more items out of a collection.